tests/models/test_models.py

```diff
@@ -5,27 +5,20 @@
 import pytest
 
 MODELS = [
-    "facebook/opt-125m",
-    "meta-llama/Llama-2-7b-hf",
-    "mistralai/Mistral-7B-v0.1",
-    "Deci/DeciLM-7b",
-    "tiiuae/falcon-7b",
+    # "facebook/opt-125m",  # Broken.
     "gpt2",
     "bigcode/tiny_starcoder_py",
-    "EleutherAI/gpt-j-6b",
     "EleutherAI/pythia-70m",
     "bigscience/bloom-560m",
-    "mosaicml/mpt-7b",
     "microsoft/phi-2",
     "stabilityai/stablelm-3b-4e1t",
-    "allenai/OLMo-1B",
+    # "allenai/OLMo-1B",  # Broken
     "bigcode/starcoder2-3b",
-    "Qwen/Qwen1.5-0.5B",
 ]
 
 
 @pytest.mark.parametrize("model", MODELS)
-@pytest.mark.parametrize("dtype", ["half"])
+@pytest.mark.parametrize("dtype", ["float"])
```

Comment:
comment that precision is needed here.

Reply:
I added assert within a test with a comment